parts/linux/cloud-init/artifacts/labels.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Update Labels for Kubernetes nodes
+After=kubelet.service
+[Service]
+Restart=always
+RestartSec=300
+EnvironmentFile=/etc/default/kubelet
+ExecStart=/bin/bash /opt/azure/containers/labels.sh
```

Comment:
nit: label-node.sh or ensure-node-labels.sh

Reply:
I didn't want to confuses this service with label-nodes service, so cut it to just labels, but would be happy to change it if we are removing the label-nodes service

Reply:
Changed to "update-node-labels.sh" due to the fact that we only run it once. I felt like the ensure was mainly if we ran it continuously. However, I could be wrong, but I expect it is possible to run the upgrade/reimage without updating the labels, so ensure could definitely work as the qualifier. I don't have a strong opinion, so am open to changing it if you still think ensure if better